### PR TITLE
Updating these with additional IPAddresses field otherwise join can b…

### PR DESCRIPTION
**Pull request by shainw**
…e incorrect, also moving comment locations and adding in missing Id.

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
 // This can potentially identify excessive traffic to a given location that could be indicative of data transfer out of your network to a group of systems on the same second level domain.  
 // For example, if one client is sending requests for test1.badguy.com and another client is sending requests for test2.badguy.com, you may not see a high enough count to be interesting. 
 // However, a combination of the requests to badguy.com could have a high enough count to be interesting. 
 // This is only Name lookups, so it would be recommended to review the Firewall\Webproxy logs in relation to the ClientIP making the interesting requests.
-
-// Data Source: DNS Events
-
+// Id: 1d9951b7-51f0-4aa7-af0c-654359aadfff
+//
+// Data Source: #DNSEvents
```
Comment (timbMSFT) [inline Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt]: Suggest using same case as acutal table name 
DnsEvents

File: Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
 // Tags: #C2, #Exfiltration
-
-DnsEvents 
-| where TimeGenerated >= startofday(ago(8d)) and TimeGenerated <= startofday(ago(1d)) //setting to 00:00:00 for the given days ago
+//
+DnsEvents
+ //setting to 00:00:00 for the given days ago
+| where TimeGenerated >= startofday(ago(8d)) and TimeGenerated <= startofday(ago(1d))
 | where SubType == "LookupQuery"
-| extend DayNumberofWeek = tostring(dayofweek(TimeGenerated)) //getting the associated number of the day of the week so we can map to a given day for later parsing if needed
```
Comment (timbMSFT) [inline Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt]: put comment on own line - cf also next one

Comment (timbMSFT) [reply Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt]: several other instances of this below

File: Hunting Queries/DnsEvents/DNS_FullNameAnomalousLookupIncrease.txt [hunk 3]
```diff
@@ -1,35 +1,43 @@
...
-// Data Source: DNS Events
-
-// Tags: #C2, #Exfiltration
-
-DnsEvents 
-| where TimeGenerated >= startofday(ago(8d)) and TimeGenerated <= startofday(ago(1d)) //setting to 00:00:00 for the given days ago
+//
+// Id: 81b478a7-46e2-4f7f-a1aa-aba5e541232b
+//
+// Data Source: #DNSEvents
```
Comment (timbMSFT) [inline Hunting Queries/DnsEvents/DNS_FullNameAnomalousLookupIncrease.txt]: let's use the actual table name:
DnsEvents

File: Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt (hunk 2 above)
Comment (shainw) [reply Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt]: Odd, not sure why they were showing as on the same line, it was not in my check in.

**Pull request merged**